esp/services/ws_machine/ws_machineService.cpp

```diff
@@ -1663,36 +1659,24 @@ void Cws_machineEx::setProcessInfo(IEspContext& context, CMachineInfoThreadParam
     //set DisplayType
     if (process.getType() && *process.getType())
     {
-        pMachineInfo->setProcessType(process.getType());
-        StringBuffer displayName;
-        getProcessDisplayName(process.getType(), displayName);
-        pMachineInfo->setDisplayType(displayName.str());
+        if (isEclAgentProcess)
+        {
+            pMachineInfo->setProcessType(eqAgentExec);
```

Comment:
@richardkchapman We want to check whether agentexec (not eclagent) is running or not. But, the pid file of the agentexec is inside the folder called eclagent. The agentexec pid file is names as eclagent.pid. So, for agentexec, I have to use process name 'eclagent' to check the agentexec pid file. After I read the pid file, at this line, I need to switch to agentexec for identifying that this is "AgentExecProcess" and, in the next line, set "Agent Exec" for display name.